yaksh/models.py

```diff
@@ -8,6 +8,12 @@
 
 
 ###############################################################################
+class Visitor(models.Model):
```

Comment:
Visitor commonly is used for a Visitor pattern, so  I would suggest a different name.